content/cli/cloud/cluster.md

```diff
@@ -4,4 +4,107 @@ menu: Cluster
 weight: 30
 ---
 
-TBD
\ No newline at end of file
+The `fluvio cloud cluster` family of commands is used to create, delete, and troubleshoot Fluvio clusters in cloud.
+
+%copy first-line%
+```bash
+$  fluvio cloud cluster -h
```

Comment:
Not sure if it matters, but better to leave just 1 space. Otherwise, the %copy% parser may choke on it.

Reply:
good catch